server/app/com/xsn/explorer/services/XSNService.scala

```diff
@@ -453,6 +459,38 @@ class XSNServiceRPCImpl @Inject()(
     result
   }
 
+  override def getMerchantnodeCount(): FutureApplicationResult[Int] = {
+    val body = s"""
+                  |{
+                  |  "jsonrpc": "1.0",
+                  |  "method": "merchantnode",
+                  |  "params": ["count"]
+                  |}
+                  |""".stripMargin
+
+    val result = retrying("masternode_count") {
+      server
+        .post(body)
+        .map { response =>
+          val maybe = getResult[Int](response)
+          maybe.getOrElse {
+            logger.debug(
+              s"Unexpected response from XSN Server, status = ${response.status}, response = ${response.body}"
```

Comment:
merchant node*, but, I think this method is not used, which we can remove

Reply:
This method is being used in the StatisticsService to get the TPoS node count.

Reply:
Count them with the in-memory store please